### PR TITLE
Adding a periodic job to move logs from GCS to IBM s3

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This Periodic job `periodic-gcs-logs-cleanup` is to move the log folders older than 7 days from GCS to IBM COS.
Currently the below three k8s periodic jobs are pushing the logs to GCS:
  - periodic-kubernetes-bazel-test-ppc64le
  - periodic-kubernetes-conformance-test-ppc64le
  - periodic-kubernetes-containerd-conformance-test-ppc64le
These jobs have an interval of `3hrs`. Hence there would be `8` entries in a day, resulting in `56` in a week's duration. The file latest-build.txt has the detail about the latest build ID.
Hence this job leaves behind `57` rows in GCS and moves the rest to IBM S3.
https://github.ibm.com/powercloud/container-dev/issues/1375

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/106#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Rajalakshmi-Girish** [comment]
\assign @mkumatag

File: config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml
Comment (mkumatag) [inline config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml:3]: This does not require the ppc64le architecture, wondering whether we can limit this job to run on the IKS cluster

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml
Comment (Rajalakshmi-Girish) [reply config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml:3]: Yes it should be fine to run on IKS cluster. I shall verify and amend.

Comment (Rajalakshmi-Girish) [reply config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml]: Yes it is working fine on IKS cluster.

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/106#issuecomment-764490043" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/106#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@Rajalakshmi-Girish: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `cleanup-gcs-logs-periodics.yaml` using file `config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/106):

>This Periodic job `periodic-gcs-logs-cleanup` is to move the log folders older than 7 days from GCS to IBM COS.
>Currently the below three k8s periodic jobs are pushing the logs to GCS:
>  - periodic-kubernetes-bazel-test-ppc64le
>  - periodic-kubernetes-conformance-test-ppc64le
>  - periodic-kubernetes-containerd-conformance-test-ppc64le
>These jobs have an interval of `3hrs`. Hence there would be `8` entries in a day, resulting in `56` in a week's duration. The file latest-build.txt has the detail about the latest build ID.
>Hence this job leaves behind `57` rows in GCS and moves the rest to IBM S3.
>https://github.ibm.com/powercloud/container-dev/issues/1375
>


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
